digitalearthau/config/products/pq_count_annual_summary.yaml

```diff
@@ -0,0 +1,37 @@
+name: pq_count_annual_summary
+description: Pixel Quality Statistics 25m
+
+metadata_type: eo
+
+metadata:
+  product_type: pq_count
+  stats:
+    period: '12m'
```

Comment:
I wonder if this may be better as a new common field such as `product_variant`, as non-stats products might have variations too? The purpose of it, as I understand, is to distinguish the variations of the `product_type` rather than actually give a precise period (eg. there may not be a whole year of data), especially in the other variants, like `'*'` ("full") below, or seasonal. The precise period is recorded in the extent start/end times.

If we *do* want to distinguish the period specifically, maybe `extent → selection_period`, or something? but that would need precise definitions of the possible values, unlike the arbitrary name allowed by a `product_variation` field.

Using a common field would also make it easier to show different product variations together on a UI without needing to know about different specific fields per product.

Reply:
I've had a similar issue with hltc and item:
```
 +    product_type: tidal_composite
 +    statistic:
 +        name: clearcount
 +        source: pq
 +        tidal_range:
 +            name: high
 +            min: 80
 +            max: 100
```

Reply:
I think whole `product_type` concept is misleading, and it doesn't help that it is not backed by any kind of table listing all valid `product_type`s. I think one should be able to just add arbitrary number of tags to any product, because that's what product type currently is: a tag, except only one tag is allowed.

Reply:
The only reason I added that field is for auto-match to work, it has no value for describing product or anything like that. For stats products period of accumulation is in the product name, but I can't add datasets by product name only by product_type + whatever other metadata.

Reply:
We discussed this in person, but I think it's worth recording here also. I don't think `product_variant` is a good idea.  In my understanding all products that share the same product type are already variants, and what distinguishes them is context specific. In the case of `nbar` differences come from the sensor/platform, in the case of statistic products it is usually period of aggregation. In some other context it might very well be some `variant` field containing a unique string (product name maybe?). Do we need to make that a "first class entity" as far as code and database is concerned? Probably not.